chingsley/ireporter
# ch(setup-db): implement connection to database

#### What does this PR do?
This pull request is deals with the configuration  and set up of a database to ensure connection, creation, and population of data in the database

#### Description of Task to be completed?
- Ensure Postgres is installed in your system
- Ensure pg admin is installed
- Using pg admin, create a database for data persistence
- execute the code in schema.js to create tables in the database

#### How should this be manually tested?
- clone the repo
- run npm install and ensure that 'make-runnable' is installed
- run the following command  in the project's root directory to create tables in the database
- cd server/src/db/schema.js createAllTables (for e.g will create all the tables)

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
#162573557

bug(validator, redflags.spec): validate users with id instead of email
#### Screenshots (if appropriate)
None

#### Questions:
None
